JavaErickHernandez/Day3/SortedList.java

```diff
@@ -0,0 +1,99 @@
+package bootcamp;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Scanner;
+
+public class SortedList {
+	static String numberOfInputs = "Enter the number of Students: ";
```

Comment:
What is the reason for not putting these variables as "final"? if you put "final" you avoid to any other person can change this variables in other par of the code.